# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../net-tools/nameif-linenum.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- net-tools-1.60/nameif.c.orig	2003-08-16 12:16:30.000000000 +0200
+++ net-tools-1.60/nameif.c	2003-08-16 12:12:55.000000000 +0200
@@ -178,7 +178,7 @@
 			continue; 
 		n = strcspn(p, " \t"); 
 		if (n > IFNAMSIZ) 
-			complain(_("interface name too long at line %d"), line);  
+			complain(_("interface name too long at line %d"), linenum);  
 		memcpy(ch->ifname, p, n); 
 		ch->ifname[n] = 0; 
 		p += n; 
